### PR TITLE
Package reorg

### PR DESCRIPTION
**Pull request by klueska**

### CODE REVIEW

**Comment by klueska** [comment]
run linux integration tests

**Comment by klueska** [comment]
run linux integration tests

File: python/lib/dcos/tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py35-syntax, py35-unit
+envlist = py36-syntax, py36-unit
```
Comment (bamarni) [inline python/lib/dcos/tox.ini]: These and below should be reverted to make Jenkins happy.

**Comment by klueska** [comment]
run linux integration tests

File: python/lib/dcos/tox.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py35-syntax, py35-unit
+envlist = {py53,py36}-syntax, {py35,py36}-unit
```
Comment (bamarni) [inline python/lib/dcos/tox.ini]: py53 -> py35, CI failed with : 
```
12:36:52 [linux-tests] ERROR: InterpreterNotFound: python3.6
12:36:52 [linux-tests] ___________________________________ summary ____________________________________
12:36:52 [linux-tests]   py53-syntax: commands succeeded
12:36:52 [linux-tests] ERROR:   py36-syntax: InterpreterNotFound: python3.6
12:36:52 [linux-tests]   py35-unit: commands succeeded
12:36:52 [linux-tests] ERROR:   py36-unit: InterpreterNotFound: python3.6
12:36:52 [linux-tests] Makefile:20: recipe for target 'test' failed
12:36:52 [linux-tests] make: *** [test] Error 1
```
Shall we move the Python 3.6 support to another PR?

**Comment by klueska** [comment]
run linux integration tests

**Comment by bamarni** [comment]
run windows integration tests

**Comment by klueska** [comment]
run linux integration tests

**Comment by bamarni** [comment]
run windows integration tests

**Comment by klueska** [comment]
run linux integration tests

**Pull request merged**